src/dbus/dlt-dbus-options.c

```diff
@@ -30,6 +30,7 @@
 
 #include <stdlib.h>
 #include <string.h>
+#include <getopt.h>
```

Comment:
This header is not really needed here. Please don't add it.

Reply:
Removed. Thanks.